index.html

```diff
@@ -66,274 +61,244 @@ <h1 class="primary-text">I'm Oscar <br> Glad to see you!</h1>
-            <li class="lang-styles">JavaScript</li>
-          </ul>
-          <button type="button" class="btn">See Projects</button>
-        </div>
-      </section>
-
-      <section class="works-temp1 works4">
-        <img src="images/SnapshootPortfolio(4).png" class="pic-temp1" alt="" />
-        <div class="works-info4">
-          <h1 class="tonic">
-            Multi-Post <br />
-            Stories
-          </h1>
-          <ul class="work-info">
-            <li class="canopy">CANOPY</li>
-            <li class="back-end-dev">
-              <img src="images/Counter(1).png" alt="" />
-              <span>Back End Dev</span>
-            </li>
-            <li class="back-end-dev">
-              <img src="images/Counter(1).png" alt="" />
-              <span>2015</span>
-            </li>
-          </ul>
-          <p class="proj-info">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="prog-lang">
-            <li class="lang-styles">HTML</li>
-            <li class="lang-styles">Css</li>
-            <li class="lang-styles">JavaScript</li>
-          </ul>
-          <button type="button" class="btn">See Projects</button>
+    
+    <div id="cards" class="container"></div>
+    <section  class="modal-container">
+      <div class="modal">
+        <h3 class="tonic" id="modal-title">Tonic</h3>
+        <span onclick="onModalClose()" class="btn-x" draggable="true"
+          >&times;</span
+        >
+        <h4>
+          CANOPY <span class="dot tag"> Back End dev </span>
+          <span class="dot tag">2015</span>
+        </h4>
+        <div class="content1">
+          <div class="image-contain fix">
+            <img
+              src="./images/work-img4.png"
+              alt="snapshoot Portfolio image"
+              class="pic-temp1"
+              id="modal-img"
+            />
+          </div>
+          <div class="stack-modal">
+            <p class="proj-info" id="modal-desc">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. A daily selection of privately personalized
+              reads; no accounts or sign-ups required. A daily selection of
+              privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <div>
+              <ul id="group-list" class="stack-icon"></ul>
+              <div class="stack-icon">
+                <a class="btn-modal" id="modal-seelive"
+                  >See live<span
+                    ><img src="./images/export.png" alt="export" /></span
+                  
+                ></a>
+                <a class="btn-modal" id="modal-seesource"
+                  >See Source<span
+                    ><img src="./images/github.png" alt="github" /></span
+                ></a>
+              </div>
+            </div>
+          </div>
```

Comment:
- [ ] Your popup windows are not well styled for some mobile phone screen sizes. Some popup windows are higher than the screen and some popup has tags that goes out of the popup. Kindly fix that.
- [OPTIONAL] On some screen sizes your popup windows has lots of blank spaces at the top and bottom. You could try to remove them so it looks better.
- [OPTIONAL] The  animation effect looks a little bit awkward. Please take a second look and try to improve it to make a better user experience.
Your design:
<img width="475" alt="image" src="https://user-images.githubusercontent.com/61919617/181590757-73f193b8-a125-4866-8fb2-5894af0f8a23.png">
<img width="480" alt="image" src="https://user-images.githubusercontent.com/61919617/181590803-870ea427-0505-4c4a-a4d8-5d4f6f25699b.png">